sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/AddressResolverExtension.java

```diff
@@ -0,0 +1,14 @@
+package com.azure.cosmos.implementation.directconnectivity;
+
+import com.azure.cosmos.implementation.RxDocumentServiceRequest;
+import com.azure.cosmos.implementation.directconnectivity.rntbd.RntbdAddressCacheToken;
+import reactor.core.publisher.Mono;
+
+import java.net.URI;
+import java.util.List;
+
+public interface AddressResolverExtension extends IAddressResolver {
```

Comment:
please document the purpose of these APIs.

Reply:
I've added a minimal description to the code on my local branch. We might elaborate and also document the IAddressResolver interface in follow-on PR.